### PR TITLE
Created UI using React-Components

### PR DESCRIPTION
**Pull request by BryteLitty**
In this PR, I did the following:
- I created a file in the components directory with the name, `Calculator.js`
- Inside the file, I created a Calculator component with the UI for the Math Magician calculator APP without math logic
- I used class-based component n creating the component
- I finally imported the `Calculator` component and used it in the `App component.

### CODE REVIEW

**Comment by prolajumokeoni** [comment]
Hi @BryteLitty, Kindly make sure that all headers of the microverse [readme](https://github.com/microverseinc/readme-template) template  is present in your readme, Please personalize the the content of the sections to be added e.g the Key feature, future features

**Review by prolajumokeoni — COMMENTED** [review]
Check summary below ⬇️

File: package.json [hunk 1]
```diff
@@ -12,10 +12,10 @@
...
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "start": "DISABLE_ESLINT_PLUGIN=true react-scripts start",
+    "build": "DISABLE_ESLINT_PLUGIN=true react-scripts build",
+    "test": "DISABLE_ESLINT_PLUGIN=true react-scripts test",
+    "eject": "DISABLE_ESLINT_PLUGIN=true react-scripts eject"
```
Comment (prolajumokeoni) [inline package.json]: - [ ] Please remove the  ```DISABLE_ESLINT_PLUGIN=true``` from the scripts this is causing an error, when I remove it  and run `npm start` your app works properly, see the video below

https://user-images.githubusercontent.com/69638013/207633824-62eab39b-b95b-45cb-a629-87d75475d74f.mp4

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @BryteLitty 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used
✔️  Good Readme



## Required Changes ♻️

- [ ] Please delete the boilerplate that got created by the CRA (texts, images, and styles), please remove images and styles not used in your app  

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by omarsalem7 — APPROVED** [review]
Hi @BryteLitty 😊, 

# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
- There is no linter error present ✔️
- The Pull Request has a proper description ✔️
- The project has met all the technical aspects you needed for this project. Keep rocking!  ✔️


Cheers and Happy coding!👏👏👏

## Optional suggestions

- [ ] [OPTIONAL] It will be better if start the commits message with a capital letter. [For more details...](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**